pom.xml

```diff
@@ -376,6 +376,58 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-pmd-plugin</artifactId>
+                <version>3.17.0</version>
+                
+
+                <executions>
+                    <!-- aggregate configuration, parent build-->
+                    <execution>
+                        <id>pmd-check-aggregate</id>
+                        <phase>validate</phase>
+                        <goals>
+                            <goal>aggregate-pmd-check</goal>
+                        </goals>
+                        <configuration>
+                            <printFailingErrors>true</printFailingErrors>
+                            <rulesets>
+                                <ruleset>${project.basedir}/pmd/legend-ruleset.xml</ruleset>
+                            </rulesets>
+                            <rules>
+	                            <requireFilesExist>
+		                            <files>
+		                                <file>${project.basedir}/pmd/LICENSE</file>
+                                    </files>
+		                        </requireFilesExist>
+                            </rules>
+                        </configuration>
+                    </execution>
+                    <!-- aggregate configuration, child build-->
+                    <execution>
+                        <id>pmd-check-child</id>
+                        <phase>validate</phase>
+                        <goals>
+                            <goal>check</goal>
+                        </goals>
+                        <configuration>
+                            <printFailingErrors>true</printFailingErrors>
+                            <rulesets>
+                                <ruleset>${project.parent.basedir}/pmd/legend-ruleset.xml</ruleset>
+                            </rulesets>
+                            <rules>
+	                            <requireFilesExist>
+		                            <files>
+		                                <file>${project.parent.basedir}/pmd/legend-ruleset.xml</file>
+                                    </files>
+		                        </requireFilesExist>
```

Comment:
Can you fix the indentation of this section to be consistent with the indentation in the rest of the file? Also, on lines 399-403 above.

Reply:
I will